auth-github/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.firebase.ui.auth.ui.github">
+
+    <application>
+
+        <activity
+            android:name=".GitHubLoginActivity"
+            android:label=""
+            android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation"
+            android:launchMode="singleTop"
+            android:theme="@style/FirebaseUI.Transparent">
+
+            <intent-filter
+                android:autoVerify="true"
+                tools:ignore="UnusedAttribute">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
+                    android:host="@string/firebase_web_host"
```

Comment:
Ok two questions about this string:

  1. I see we have duplicate `strings.xml` entries for `firebase_web_host` in both `auth/.../strings.xml` and `auth-github/.../strings.xml` ... is there a reason it needs to exist outside of the GitHub module?  I feel like we only use it there but could be forgetting something.
  1. Now that we're sure you want to use GitHub auth when you include the `-github` module, should we explode at runtime if this string is still equal to `CHANGE-ME`?  I think that might be helpful.

Reply:
Yup, both those birds were killed with the same stone in the original implementation 😁: https://github.com/firebase/FirebaseUI-Android/blob/f83c6219a32456b24c9a78cd597f97943c938669/auth/src/main/java/com/firebase/ui/auth/AuthUI.java#L973-L977

1. We need `firebase_web_host` in the auth module to do that check
1. We do that check 😂

Reply:
D'oh